src/constants/indicators.js

```diff
@@ -171,7 +171,7 @@ export default {
   },
   dispersion: {
     name: 'Local governments per 100k people',
-    format: FORMATS.PERCENT_ZERO_OR_ONE_DECIMAL,
+    format: FORMATS.ZERO_OR_ONE_DECIMAL,
```

Comment:
I've tried to get three significant figures everywhere. So for indicators mostly under 10, two decimal places, and for indicators mostly over 10, one decimal place. So this can be `ZERO_TO_TWO_DECIMALS` or `ONE_OR_TWO_DECIMALS`. Either is fine, but if you do `ONE_OR_TWO_DECIMALS`, the y axis to the chart looks a bit neater because each label will be the same length.

![image](https://user-images.githubusercontent.com/4443482/33290632-68ae47a4-d417-11e7-9282-4b3203437915.png)

vs

![image](https://user-images.githubusercontent.com/4443482/33290644-754439b0-d417-11e7-8aa4-ae4ba4831285.png)

Reply:
Yes, I like it.